qt/python/mantidqt/mantidqt/plotting/figuretype.py

```diff
@@ -76,9 +76,7 @@ def axes_type(ax):
         elif any(isinstance(col, Line3DCollection) for col in ax.collections):
             axtype = FigureType.Wireframe
     elif len(ax.images) > 0 or len(ax.collections) > 0:
-        if any(isinstance(col, LineCollection) for col in ax.collections):
-            axtype = FigureType.Contour
-        elif any(isinstance(col, PathCollection) for col in ax.collections):
+        if any(isinstance(col, ContourSet) for col in ax.collections):
```

Comment:
The object returned by `Contour` was a custom object holding multiple `Collections` and thus not an Artist.
In version 3.8 however, it is an Artist. So it caused an error because the return type has changed. (I forgot to take notes so I unfortunately do not remember the exact error, but I think it was an `OutOfRange` error). We can use `ContourSet` to get the contour now because it's instead a plain `Collection` which contains a single path per contour level.